lang/python/python.talon

```diff
@@ -38,7 +38,7 @@ self taught: "self."
 pie test: "pytest"
 state past: "pass"
 
-raise {user.python_exception}: user.insert_cursor("raise {python_exception}([|])")
+raise {user.python_exception}: user.insert_between("raise {python_exception}(", ")")
```

Comment:
Would it be worth just dropping the parens for this? Sometimes I do want to just `raise NotImplementedError` for example without the visual noise.

Reply:
Maybe? I don't want to change any actual behavior with this PR though. Submit a PR if you want.